src/Specifications.md

```diff
@@ -1,17 +1,71 @@
-Video RAM       | 8 KB                           |                 | 16 KB           |                |
-Screen Size     | LCD 4,7 x 4,3 cm               | LCD 4,8 x 4,4 cm| TFT 4,4 x 4 cm  |                |
-Resolution      | 160x144                        |                 |                 | 256x224 (including border)|
-Sprites         | Max 40 per screen, 10 per line |                 |                 |                |
-Palettes        | 1x4 BG, 2x3 OBJ                |                 | 8x4 BG, 8x3 OBJ | 1+4x3, 4x15 (border) |
-Colors          | 4 grayshades                   |                 | 32768 colors    |                |
-Horizontal Sync | 9.198 KHz                      |                 |                 | 9.41986 KHz    |
-Vertical Sync   | 59.73 Hz                       |                 |                 | 61.1679 Hz     |
-Sound           | 4 channels with stereo sound   |                 |                 |                |
-Power           | DC6V 0.7W                      | DC3V 0.7W       | DC3V 0.6W       |                |
+<style>
+td {
+    text-align: center;
+}
+td:first-child {
+    text-align: left;
+}
+</style>
+
+<table>
+  <thead>
+    <tr>
+      <th></th><th>Game Boy (DMG)</th><th>Game Boy Pocket (MGB)</th><th>Super Game Boy (SGB)</th><th>Game Boy Color (CGB)</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td>CPU</td><td colspan="4">8-bit 8080-like Sharp CPU (speculated to be a SM83 core)</td>
+    </tr>
+    <tr>
+      <td>CPU freq</td><td colspan="2">4.194304&nbsp;MHz</td><td>Depends on revision<sup class="footnote-reference"><a href="#sgb_clk">1</a></sup></td><td>Up to 8.388608&nbsp;MHz</td>
+    </tr>
+    <tr>
+        <td>Work RAM</td><td colspan="3">8&nbsp;KiB</td><td>32&nbsp;KiB<sup class="footnote-reference"><a href="#compat">2</a></sup> (4&nbsp;+&nbsp;7&nbsp;×&nbsp;4&nbsp;KiB)</td>
+    </tr>
+    <tr>
+        <td>Video RAM</td><td colspan="3">8&nbsp;KiB</td><td>16&nbsp;KiB<sup class="footnote-reference"><a href="#compat">2</a></sup> (2&nbsp;×&nbsp;8&nbsp;KiB)</td>
+    </tr>
+    <tr>
+        <td>Screen</td><td>LCD 4.7&nbsp;×&nbsp;4.3&nbsp;cm</td><td>LCD 4.8&nbsp;×&nbsp;4.4&nbsp;cm</td><td>CRT TV</td><td>TFT 4.4&nbsp;×&nbsp;4&nbsp;cm</td>
+    </tr>
+    <tr>
+        <td>Resolution</td><td colspan="2">160&nbsp;×&nbsp;144</td><td>160&nbsp;×&nbsp;144 within 256&nbsp;×&nbsp;224 border</td><td>160&nbsp;×&nbsp;144</td>
+    </tr>
+    <tr>
+        <td>OBJ ("sprites")</td><td colspan="4">8&nbsp;×&nbsp;8 or 8&nbsp;×16 ; max 40 per screen, 10 per line</td>
+    </tr>
+    <tr>
+        <td>Palettes</td><td colspan="2">BG: 1&nbsp;×&nbsp;4, OBJ: 2&nbsp;×&nbsp;3</td><td>BG/OBJ: 1&nbsp;+&nbsp;4&nbsp;×&nbsp;3, border: 4&nbsp;×&nbsp;15</td><td>BG: 8&nbsp;×&nbsp;4, OBJ: 8&nbsp;×&nbsp;3<sup class="footnote-reference"><a href="#compat">2</a></sup></td>
+    </tr>
+    <tr>
+        <td>Colors</td><td>4 shades of green</td><td>4 shades of gray</td><td colspan="2">32768 colors (RGB555)</td>
```

Comment:
Why was RGB555 added here? I don't feel it's relevant - it's a "format", but from a specification point of view 32768 colors with 1/3 of the bits per color is pretty standard and 32768 colors on RGB gives already this information

Reply:
I think "RGB555" is most precise compared to "32768 colors", "15-bit color", "high color", etc. It clearly says that all channels are equal 5-bit, not a 15-bit palette index or something weird like RGB465.

Reply:
Yes, this was my line of thinking.

Reply:
I don't think is most precise. On the contrary it's confusing because specifying a particular palette or encoding implies you are trying to communicate something non-standard, while actually, by standard (if not otherwise stated), every channel is encoded by the same amount of information: 32768 colors need 2^15 bits, I don't see why I should think each channel is not getting encoded by (15/3) bits anyway

Reply:
There are some pretty unusual color encodings in https://en.wikipedia.org/wiki/List_of_video_game_console_palettes, including for the NES, which don't just divide the bits into equal RGB (nor even YUV, HSL, etc) components.

Reply:
The fact you define them "unusual" proves my point. You mention the encoding when it's unusual and it what way, not when it's standard.
In the exact same page you linked, under "Super NES", you can read "15-bit RGB (32,768 color)" for what we are discussing here.

Reply:
"32768 colors (15-bit)" then?

Reply:
You are suggesting something completely unrelated to what we are discussing now. ISSO slipt information about how the channels are encoded, you now want to enforce (re-explain) that to encode 32768 colors you need 15 bits

Reply:
Which for me is again wrong. The wikipedia page is not a manual and the target is different. This is an introductory specification table aimed to give an *overview* in a technical document.
Why are you feeling additional information is needed? What information you can't get from "32768 colors (RGB)"?

Reply:
What I meant is, if you're against the extra information, just go ahead and say "32,768 colors" because "(RGB)" doesn't clarify anything. I was suggesting "15-bit" or "15-bit RGB" as alternatives to "RGB555"; IMO any of those would clarify *how* those 32,768 colors are determined.

Reply:
Imo "32768 RGB colors" without further precision strikes a good deal between disambiguation ("it's RGB, not YUV or a fixed palette") and terseness ("32768 → 15-bit, if it's RGB and no precisions are given, it's likely RGB555").

Reply:
I've been talked in how "unusual" this can result and how it can be useful anyway to specify RGB555 so `32,768 colors (15 bit RGB)` is okay for me but if you really think this should go on a introductory specification table I will accept `32,768 (RGB555)`, too

Reply:
I think "15-bit RGB" strikes the best balance we all agree on, so let's use that. Thanks!